src/Core/Framework/Demodata/Generator/CategoryGenerator.php

```diff
@@ -95,6 +95,7 @@ private function createCategory(DemodataContext $context, array $pageIds, array
             'mediaId' => $context->getRandomId('media'),
             'description' => $context->getFaker()->text(),
             'tags' => $this->getTags($tags),
+            'customFields' => ['demodata' => true],
```

Comment:
should we prefix this with `shopware` or so? and the key should be put into a global constant